multidim-interop/README.md

```diff
@@ -0,0 +1,58 @@
+# Interoperability test
+
+This tests that different libp2p implementations can communicate with each other
+on each of their supported capabilites.
+
+Each version of libp2p is defined in `versions.ts`. There the version defines
+its capabilities along with the id of its container image.
+
+# Test spec
+
+The implementation is run in a container and is passed parameters via
+environment variables. The current parameters are:
+
+| Name         | Description                                                  | Is Optional                                                     |
+| ------------ | ------------------------------------------------------------ | --------------------------------------------------------------- |
+| REDIS_ADDR   | A different address to connect to redis (default redis:6379) | yes                                                             |
+| ip           | IP address to bind the listener to                           | no                                                              |
+| transport    | The transport to use                                         | no                                                              |
```

Comment:
Is it useful to define the valid values for "transport"?  (same goes for "muxer" and "security").

Do we need to define the behavior if it gets pass a "transport" that it doesn't have a mapping for?

Reply:
Valid values of transport depend on what the implementation says it supports. e.g. if an implementation doesn't support TCP, it'll never be called with a `transport=tcp` param.